test/services/ReturnsServiceSpec.scala

```diff
@@ -156,6 +147,32 @@ class ReturnsServiceSpec extends VatRegSpec with FeatureSwitching {
       await(service.saveTurnover(turnoverEstimate = testTurnover)) mustBe returns
     }
 
+    "save a complete model and remove FRS if above 150000" in new Setup {
```

Comment:
hey! small one. should we have another test here for exemption ineligibility check? thanks
https://github.com/hmrc/vat-registration-frontend/blob/033e4f798a87e031f56d8813f7e5378500a05e2c/app/services/ReturnsService.scala#L97